ontologies/Schema/resultlist.ttl

```diff
@@ -0,0 +1,37 @@
+@prefix wresultlist:  <http://wellcomecollection.org/ontologies/resultlist/> .
+@prefix xsd:          <http://www.w3.org/2001/XMLSchema#> .
+@prefix rdf:          <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix dcterms:      <http://purl.org/dc/terms/> .
+@prefix owl:          <http://www.w3.org/2002/07/owl#> .
+@prefix rdfs:         <http://www.w3.org/2000/01/rdf-schema#> .
+
+<http://wellcomecollection.org/ontologies/resultlist>
+    rdf:type owl:Ontology ;
+    dcterms:title "Result list ontology"@en ;
+    dcterms:created "2017-11-07"^^xsd:date ;
+    dcterms:description "Ontology describing the properties of a result list"@en .
+
+wresultlist:ResultList rdf:type owl:Class ;
+    rdfs:label "Result list"@en ;
+    rdfs:comment "A collection of works."@en ;
```

Comment:
Only used for works?
Why not a WorksList then?

Reply:
Its a good point. Would we use result lists for lists of items, holdings and potentially concepts in the future. @jamesgorrie I will pick up with platform team tomorrow.